.gitpod.yml

```diff
@@ -32,6 +32,8 @@ tasks:
       make run-server
       gp ports await 8065
       make stop-server
+      make test-data
```

Comment:
0/5 does the test data make it easier to develop?

Reply:
I'm not sure if we should add this. @sinansonmez What do you think?

Reply:
I think we should definetely add it. Because everytime I create a new workspace this is the first thing I do.